src/backend/metal/src/command.rs

```diff
@@ -821,11 +821,11 @@ impl CommandSink {
     ) where
         I: Iterator<Item = soft::RenderCommand<&'a soft::Own>>,
     {
+        //assert!(AutoReleasePool::is_active());
```

Comment:
nit: remove this?

Reply:
this is intentional - we need to make sure there is an active autorelease pool set up by the caller here, but I don't know how to code it yet